src/main.cpp

```diff
@@ -106,6 +119,17 @@ void setup() {
     SOFTWARE_VERSION_MAJOR, SOFTWARE_VERSION_MINOR, SOFTWARE_VERSION_PATCH,
     HARDWARE_VERSION_MAJOR, HARDWARE_VERSION_MINOR);
   Logger::notice("rESCue", buf);
+
+#ifdef WIFI_ENABLED
+  WiFi.mode(WIFI_STA);
+  WiFi.begin("INSERT_YOUR_WIFI_SSID_HERE", "INSERT_YOUR_WIFI_PASSWORD_HERE");
```

Comment:
**Suggestion:** Avoid hardcoding WiFi credentials in the source code. This is a security risk and also reduces the flexibility of the code. Instead, consider implementing a mechanism to input these credentials at runtime or from a secure and separate configuration file.
```suggestion
#ifdef WIFI_ENABLED
  WiFi.mode(WIFI_STA);
  WiFi.begin(WIFI_SSID, WIFI_PASS);
#endif // WIFI_ENABLED
```